packages/yourloops/pages/notifications/notification.tsx

```diff
@@ -268,34 +260,34 @@ export const Notification = (props: NotificationProps): JSX.Element => {
       notifications.update()
     }
   }
-
-  const closeTeamAcceptDialog = (teamId?: string): void => {
+  const joinTeam = async (): Promise<void> => {
+    await acceptInvitation()
+    onCloseDialog()
+  }
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Need to add a try/catch here to handle error if `acceptInvitation` fails. However the user isn't notified if there's in an error and the button stays on loading.